diff --git a/dist/cjs/index.cjs b/dist/cjs/index.cjs
index df729920a3917c3d6b6ee6c464129a2d80c6686f..f60c9ee1c237e5f6c65f13fea8f2c2d00103bb06 100644
--- a/dist/cjs/index.cjs
+++ b/dist/cjs/index.cjs
@@ -8651,16 +8651,6 @@ function getDefaultBabelOptions({ browserslist, transpilerOptions }) {
                 ])
         ],
         plugins: [
-            // If preset-env will be included, shipped proposals will be included already. If not, apply them here
-            ...(includePresetEnv
-                ? []
-                : [
-                    resolveModule("@babel/plugin-proposal-object-rest-spread"),
-                    resolveModule("@babel/plugin-proposal-async-generator-functions"),
-                    resolveModule("@babel/plugin-proposal-optional-catch-binding"),
-                    resolveModule("@babel/plugin-proposal-unicode-property-regex"),
-                    resolveModule("@babel/plugin-proposal-json-strings")
-                ]),
             // Force the use of helpers (e.g. the runtime). But *don't* apply polyfills.
             [
                 resolveModule("@babel/plugin-transform-runtime"),
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 5a415c123b26ac92639cbff17e549b385b8b3566..9f7ce09c794257f7acafb7af866a2acb9375ae83 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -8649,16 +8649,6 @@ function getDefaultBabelOptions({ browserslist, transpilerOptions }) {
                 ])
         ],
         plugins: [
-            // If preset-env will be included, shipped proposals will be included already. If not, apply them here
-            ...(includePresetEnv
-                ? []
-                : [
-                    resolveModule("@babel/plugin-proposal-object-rest-spread"),
-                    resolveModule("@babel/plugin-proposal-async-generator-functions"),
-                    resolveModule("@babel/plugin-proposal-optional-catch-binding"),
-                    resolveModule("@babel/plugin-proposal-unicode-property-regex"),
-                    resolveModule("@babel/plugin-proposal-json-strings")
-                ]),
             // Force the use of helpers (e.g. the runtime). But *don't* apply polyfills.
             [
                 resolveModule("@babel/plugin-transform-runtime"),